src/usr/local/buildpack/utils/linking.sh

```diff
@@ -3,7 +3,7 @@
 # use this if custom env is required, creates a shell wrapper to /usr/local/bin
 function shell_wrapper () {
   local FILE
-  FILE="$(get_bin_path)/bin/${1}"
+  FILE="$(get_bin_path)/${1}"
```

Comment:
Why this change?

Reply:
because the `bin` in the path is already returned by `get_bin_path`
I checked the usages, apparently nothing uses this function yet, that is why it did not broke.

Reply:
I introduced the error in this PR: https://github.com/containerbase/buildpack/pull/275/files#diff-5a9264b9dd9a3e58f84880c628edc05ab79a00476745f5836ba8d2123254f26eL6